install/kubernetes/helm/istio/templates/crds.yaml

```diff
@@ -99,6 +99,30 @@ spec:
 ---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
+metadata:
+  name: sidecars.networking.istio.io
+  annotations:
+    "helm.sh/hook": crd-install
```

Comment:
I think the agreement is to stop using the hook, it creates a lot of problems.

Reply:
I have no idea. I am simply copy pasting to add a new CRD. give me a patch to add and I will replace this blob with it.

Reply:
Just remove the annotation. 
@sdake  has been working on this - not sure if the new CRDs need to go to a different file.
Needs to be tested in the various modes.

Reply:
@rshriram CRDs for 1.1 go in the `install/kubernetes/helm/istio/files/crd-11.yaml` file.